util.py

```diff
@@ -419,6 +419,86 @@ def winedll_override(dll: str, dtype: Literal['n', 'b', 'n,b', 'b,n', '']) -> No
     )
 
 
+def patch_libcuda() -> bool:
+    """Patches libcuda to work around games that crash when initializing libcuda and are using DLSS.
+
+    We will replace specific bytes in the original libcuda.so binary to increase the allowed memory allocation area.
+
+    The patched library is overwritten at every launch and is placed in .cache
+
+    Returns true if the library was patched correctly. Otherwise returns false
+    """
+    cache_dir = os.path.expanduser('~/.cache/protonfixes')
+    os.makedirs(cache_dir, exist_ok=True)
+
+    try:
+        # Use ldconfig -p to search library paths
+        try:
+            output = subprocess.check_output(['ldconfig', '-p'], stderr=subprocess.STDOUT, text=True)
```

Comment:
Here, I would use either `subprocess.run` or `subprocess.Popen` to create the subprocess  so we can have some guarantee of the text returned from the command. This can be a problem, especially when running outside the Steam Runtime, as character encoding is currently platform/host dependent until Python 3.15.

See https://github.com/python/cpython/blob/main/Lib/ctypes/util.py#L301

Reply:
I would maybe specify the absolute path to the executable too by either hardcoding the path or use` shutil.which` to find it. I think it would be acceptable to hardcode `/sbin/ldconfig` as the standard library does it, which signals to me that it's stable and a fundamental tool. Same with the Steam Runtime code, and the Steam client's `run.sh` script at `$HOME/.local/share/Steam/ubuntu12_32/steam-runtime`

Reply:
This should be resolved with https://github.com/Open-Wine-Components/umu-protonfixes/pull/136/commits/338d9e5fb36625015196a5a9c306d48fc278d7bc

Used `subprocess.Popen` as CI doesn't like `subprocess.run` apparently.